python/ray/data/_internal/planner/exchange/fast_repartition_task_scheduler.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ray.data._internal.execution.interfaces import RefBundle
+from ray.data._internal.planner.exchange.interfaces import ExchangeTaskScheduler
+from ray.data._internal.progress_bar import ProgressBar
+from ray.data._internal.remote_fn import cached_remote_fn
+from ray.data._internal.split import _split_at_indices
+from ray.data._internal.stats import StatsDict
+from ray.data.block import Block, BlockMetadata
+from ray.types import ObjectRef
+
+
+class FastRepartitionTaskScheduler(ExchangeTaskScheduler):
```

Comment:
How about we call it `SplitRepartitionTaskScheduler` to be more specific? Fast is kind of vague to me.